aiocache/backends/memcached.py

```diff
@@ -1,13 +1,13 @@
 import asyncio
-from typing import Optional
+from typing import Any, Optional
 
 import aiomcache
 
 from aiocache.base import BaseCache
 from aiocache.serializers import JsonSerializer
 
 
-class MemcachedBackend(BaseCache[bytes]):
+class MemcachedBackend(BaseCache[bytes, Any]):
```

Comment:
I think all these need to be defined as a TypeVar. Then the user would need to annotate their code as: `MemcachedCache[Any]` and similar.

Though, probably better to get #684 done before this one.